tests/src/Interop/MarshalAPI/ReadWrite/ReadWriteObject.cs

```diff
@@ -0,0 +1,119 @@
+﻿using System;
```

Comment:
I don't think we have a policy in place.  In general for the more 'runtime'-y features such as interop it makes sense to put the tests in CoreCLR. Marshal APIs are sort like a gray area that can land either way. My preference is to put them in CoreCLR tests unless it's super clear they belong in CoreFX (such as in System.Runtime.InteropServices.dll, etc). Moving forward, I think we'd want CoreCLR tests to run on multiple runtime configuration as well (they are runtime tests that applies everywhere, after all)